v2/index.html

```diff
@@ -1810,28 +1810,21 @@ <h5>Tremolo, Slash</h5>
         <section>
             <h4>Shortcuts</h4>
             <section>
-                <h5>Repeated figures</h5>
-                <table class="simple">
-                    <thead>
-                    <tr>
-                        <th>Code</th>
-                        <th>Name</th>
-                    </tr>
-                    </thead>
-                    <tbody>
-                    <tr>
-                        <td><code>!</code></td>
-                        <td>beginning and end of passage</td>
-                    </tr>
-                    <tr>
-                        <td><code>f</code></td>
-                        <td>repetition indication of the notes that appear within !...!</td>
-                    </tr>
-                    </tbody>
-                </table>
+                <h5>Repeat group</h5>
+                <p>
+                    If one or notes or rests are repeated several times, a repeat group
+                    MAY be used.
+                </p>
+                <p>
+                    A repeat group is only valid within a single measure. To repeat entire
+                    measures, use the <a>Measure repeat</a> shortcut.
```

Comment:
```suggestion
                    A repeat group is only valid within a single measure.
```
The repetition of an entire measure has nothing to do with repeat groups and it makes it very confusing to say this here.